src/index.html

```diff
@@ -10,8 +10,212 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
recheck formatting everywhere
https://mate-academy.github.io/style-guides/htmlcss.html